src/app/public/modules/phone-field/phone-field.component.scss

```diff
@@ -0,0 +1,41 @@
+@import "~@skyux/theme/scss/mixins";
```

Comment:
Could we disable the left-hand border of the input (the border next to the dropdown) so that it doesn't appear as a 2px-wide border?

**Before:**
![Screen Shot 2019-04-11 at 3 28 49 PM](https://user-images.githubusercontent.com/12497062/55986965-87710600-5c6e-11e9-9783-9b661b94c222.png)

**After:**
![Screen Shot 2019-04-11 at 3 29 14 PM](https://user-images.githubusercontent.com/12497062/55987004-98ba1280-5c6e-11e9-85f6-1090a5807801.png)

Reply:
Yup. Done